### PR TITLE
adds menu hover links bb

### PR DESCRIPTION
**Pull request by rafibomb**

### CODE REVIEW

File: src/building-blocks/menu-hover-lines/menu-hover-lines.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.menu-hover-lines li a {
+  padding: 0.75rem 0;
+  color: rgba($body-font-color, 0.5);
+  position: relative;
+}
+
+.menu-hover-lines li a {
+  margin-left: 1rem;
```
Comment (canterberrie) [inline src/building-blocks/menu-hover-lines/menu-hover-lines.scss]: You're using `.menu-hover-lines li` a twice here

Comment (canterberrie) [inline src/building-blocks/menu-hover-lines/menu-hover-lines.scss]: also, why not nest `li` inside of `.menu-hover-lines`?

File: src/building-blocks/menu-hover-lines/menu-hover-lines.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.menu-hover-lines a:before {
+  top: 0;
+  left: 0;
+}
+
+.menu-hover-lines a:after {
+  bottom: 0;
+  right: 0;
```
Comment (canterberrie) [inline src/building-blocks/menu-hover-lines/menu-hover-lines.scss]: feel like we can be nesting all of these selectors?

File: src/building-blocks/menu-hover-lines/menu-hover-lines.scss (hunk 1 above)
Comment (canterberrie) [inline src/building-blocks/menu-hover-lines/menu-hover-lines.scss]: `.menu-hover-lines li a` is repeated. move this into the above CSS block

**Comment by IamManchanda** [comment]
@rafibomb sorry but as a sass purist .... this should go something like this below 
aka proper nesting!

```scss
.menu-hover-lines {
  // Your code here
  li {
    a {
      // Your code here
    }
    &:first-child a {
      // Your code here
    }
    &.active > a {
      // Your code here
    }
  }
}
```

**Comment by rafibomb** [comment]
@IamManchanda and @canterberrie True - though the code output is the same. It makes no difference in this case in terms of the final code since the selectors will look the same.

**Comment by IamManchanda** [comment]
http://thesassway.com/advanced/modular-css-naming-conventions

**Comment by rafibomb** [comment]
Haha love the dedication to this! I've updated to code :)

**Comment by canterberrie** [comment]
@IamManchanda we won! thanks Rafi! Merging now.

**Comment by IamManchanda** [comment]
Yup High Five Christine !

**Pull request merged**